vms/platformvm/txs/executor/camino_tx_executor_test.go

```diff
@@ -1834,7 +1838,33 @@ func TestAddAddressStateTxExecutor(t *testing.T) {
 			txFlag:        txs.AddressStateBitKYCVerified,
 			existingState: txs.AddressStateEmpty,
 			remove:        true,
-			expectedErr:   errAddrStateNotPermitted,
+			expectedErrs:  []error{errAddrStateNotPermitted, errAddrStateNotPermitted},
+		},
+		// Bob has KYC role, and he is trying to give Alice KYC Expired state
+		"Upgrade: 1, State: KYC, Flag: KYC Expired, Add, Different Address": {
+			UpgradeVersion: 1,
+			stateAddress:   bob,
+			targetAddress:  alice,
+			txFlag:         txs.AddressStateBitKYCExpired,
+			existingState:  txs.AddressStateRoleKYC,
+			expectedState:  txs.AddressStateKYCExpired,
+			expectedErrs:   []error{nil, errNotSunrisePhase1},
+			remove:         false,
+			executor:       bob,
+			executorAuth:   &secp256k1fx.Input{SigIndices: []uint32{0}},
+		},
+		// Bob has KYC role, alice tries to executor her KYC Expired state
+		"Upgrade: 1, State: KYC, Flag: KYC Expired, wrong executor": {
+			UpgradeVersion: 1,
+			stateAddress:   bob,
+			targetAddress:  alice,
+			txFlag:         txs.AddressStateBitKYCExpired,
+			existingState:  txs.AddressStateRoleKYC,
+			expectedState:  txs.AddressStateKYCExpired,
+			expectedErrs:   []error{errSignatureMissing, errNotSunrisePhase1},
```

Comment:
errors order seems wrong, if I'm right, test isn't working cause both time its sp0
upd: just tested, `chaintime` is `time.Time(1997-01-01T05:00:00+05:00)` and `sp1` ts is `time.Time(0001-01-01T00:00:00Z)`